testing/terraform-ech/go-ech/ech.go

```diff
@@ -0,0 +1,128 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
It's a bit strange to me that this package is under the `terraform-ech` folder, considering it doesn't have anything to do with Terraform and the functions it exports are only called from the smoke test code.  Could we find a more relevant location for it in the codebase?